workshops/py_Christmas/README.md

```diff
@@ -0,0 +1,381 @@
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range (5):
+        turtle.forward(size)
+        turtle.right(144)
+        turtle.forward(size)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+```
+I guess you would've understood atleast 90% of the above code easily. I will explain the new stuff.
+The ******size****** parameter decides the legth of the edges of the star.
+Now we move the turtle forward by a distance same as that of the size mentioned. Now we turn the turtle to right by 144 degreesand move it by the same distance. Repeate the step 5 times and you will have a star. Got it ? 
+
+Last function in this file to make the tiny balls on either sides of the boxes (green part) pf the christmas tree.
+Go through the code. You probably will understand it pretty easily.
+```python
+def balls(turtle, color, x, y, radius):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    turtle.circle(radius)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+``` 
+Yes, ******turtle****** library has a circle function. Made our job pretty easy, didn't it ? 
+
+So yeah, that's it ... our funtions to make the figures is ready. Wanna see the whole code again ? Take a look. 
+
+```python
+import turtle
+def box1 (turtle, color, x, y, width, height):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x,y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range(2):
+        turtle.forward(width)
+        turtle.left(90)
+        turtle.forward(height)
+        turtle.left(90)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+
+
+def star(turtle, color, x, y, size):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range (5):
+        turtle.forward(size)
+        turtle.right(144)
+        turtle.forward(size)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+
+def balls(turtle, color, x, y, radius):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    turtle.circle(radius)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+``` 
+All good ? 
+
+# Now, lets move on to Next file : ChristmasTree.py
+
+
+Now open the main.py 
+
+In this page we are gonna make a tree stem made of rectangle then we are gonna make multiple rectangles stacked above each other to make it look like a christmas tree. 
+Shall we begin ?
+
+```python
+from turtle import *
+from Figures import *
+from random import randint
+
+```
+Here, we import all fuctions from 'turtle', all functions from 'Figures' and finally 'randint' function from 'random' so that we can use it to generate a random integer.
+Why do we need a random integer ? We'll see .
```

Comment:
```suggestion
Here, we import all functions from 'turtle', all functions from 'Figures', and finally 'randint' function from 'random' so that we can use it to generate a random integer.
Why do we need a random integer? We'll see.
```